docs/fundamentals/external-parameters.md

```diff
@@ -53,6 +53,31 @@ var value = builder.Configuration[key]; // value = "local-value"
 > [!IMPORTANT]
 > However, you don't need to access this configuration value yourself in the app host. Instead, the <xref:Aspire.Hosting.ApplicationModel.ParameterResource> is used to pass the parameter value to dependent resources. Most often as an environment variable.
 
+### Prompt for parameter values in the dashboard
+
+From .NET Aspire version 9.4 onwards, if your code adds parameters but doesn't set them, then you will see a prompt to configure their values in the .NET Aspire dashboard. The **Unresolved parameters** message is displayed and you can click **Enter values** to resolve the problem:
```

Comment:
While it's true that this is added in 9.4, I'm trying to avoid mentioning versions outside of what's new. Since we're trying to keep everyone up-to-date with the latest, let's drop the version here.

```suggestion
If your code adds parameters but doesn't set them, you'll see a prompt to configure their values in the .NET Aspire dashboard. The **Unresolved parameters** message appears, and you can select **Enter values** to resolve the problem:
```